user_experience_working_group/README.md

```diff
@@ -30,29 +30,6 @@ Use the Crescendo Release grow Flow's developer base and network activity
 KR1: Acquire new interest on Flow using the Crescendo Release
 KR2: Activate new developers to interact with the Crescendo
 
-## **** Agenda ****

```

Comment:
@Aliserag moved this into the meeting file. The idea is to have the agenda items listed in the meeting file before every meeting.